environment-cpu-linux.yml

```diff
@@ -8,15 +8,16 @@ dependencies:
   - python=3.7
   - numpy=1.16
   - scipy=1.3
-  - pytorch=1.2.0
+  - pytorch=1.5
   - cpuonly
   - tqdm=4.32
   - scikit-learn=0.21
   - ffmpeg
   - resampy
+  - torchaudio=0.5.0
   - pip
   - pip:
     - musdb==0.3.1
-    - norbert==0.2.0
+    - norbert>=0.2.0
```

Comment:
do we need a newer version? if yes, lets freeze that

Reply:
actually we don't need norbert anymore, let's remove it

Reply:
yes